src/lib/babel-config.js

```diff
@@ -4,12 +4,8 @@ export default (env, options={}) => ({
 		[require.resolve('babel-preset-env'), {
 			loose: true,
 			modules: options.modules || false,
+			browsers: options.browsers,
```

Comment:
Oh, haha, great!

I'm thinking now, to answer my own question (below), that I need to respect any `"babel"` config inside the `package.json`.

With #56, how will users customize their babel options? I should account for that too.